internal/mempool/v1/mempool.go

```diff
@@ -443,6 +448,9 @@ func (txmp *TxMempool) Update(
 		}
 	}
 
+	// TODO: Purge transactions from height and timestamp indexes that have
```

Comment:
Any idea how risky it is to leave these around indefinitely?

Reply:
You can't leave them around. One, it would grow indefinitely, and two, we need to know what to remove from the mempool.